src/Host.cpp

```diff
@@ -534,6 +534,20 @@ QString Host::getMmpMapLocation() const
 {
     return mpMap->getMmpMapLocation();
 }
+// error and debug consoles inherit font of the main console
```

Comment:
For the error console I think this is fine. However, if multi-playing, does this mean that when ever the user changes the preferences for one profile the debug console will switch to the font for the main console for THAT profile and then change again should the user change the setting to have a different font for the main console in *another* active profile?

Reply:
That's correct., and I've tried it in practice - it feels okay this way.

Reply:
I  cannot work it out - is it possible to use different fonts for the Qt Widget at the bottom of the `TTextEdit` for difference pieces of text shown in that widget or is a single font setting applied across *all* content? The reason I ask is that it might lend itself to *part* of the issue of resolving the source (profile involved) for each line of content in the Central Debug Console when multi-playing.

Reply:
I am not sure what are you asking, can you rephase?